5-infrastructure/modules/base-env/README.md

```diff
@@ -0,0 +1,153 @@
+# Base Env Module
+This module creates the resources needed for the BOA Deloyment.
+
```

Comment:
We should have some more info here since this composes the environment. Let's call out key resources deployed, important inputs like project ids etc. This way we can keep the docs for each env lighter.